prime-router/metadata/hl7_mapping/common/observation-result.yml

```diff
@@ -0,0 +1,195 @@
+elements:
+  - name: result-set-id
+    value: [ '%resultIndex + 1' ]
+    hl7Spec: [ '%{hl7OBXField}-1' ]
+
+  - name: result-value-type-st
+    condition: '%resource.value is String or (%resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").exists() and %resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").code = "unknown")'
+    value: [ '"ST"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-dt
+    condition: '%resource.value is dateTime'
+    value: [ '"DT"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-ce
+    condition: '%resource.value is CodeableConcept'
+    # Note that for v.2.5 and earlier CWE is not in-spec.
+    value: [ '"CE"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-cwe
+    resource: '%resource.value'
+    condition: '%resource is CodeableConcept and (%resource.coding[0].version.exists() or %resource.coding[1].version.exists() or %resource.text.exists())'
+    # Note that for v.2.5 and earlier, CWE is not officially in-spec, but is widely used.
+    value: [ '"CWE"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-sn
+    condition: '%resource.value is Ratio'
+    value: [ '"SN"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-nm
+    condition: '%resource.value is Quantity'
+    value: [ '"NM"' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: observation-identifier-code
```

Comment:
Before, there were two of the same name that had a condition for when the value existed and one for when the value did not. Both pointed to the same schema. I took out the condition and left only this one pointing to the schema. All tests pass in my environment.